### PR TITLE
Fix user permissisons on index cache directory

### PR DESCRIPTION
**Pull request by yashvardhannanavati**

### CODE REVIEW

File: iib/workers/tasks/opm_operations.py [hunk 1]
```diff
@@ -427,7 +427,7 @@ def verify_cache_insertion_edit_dockerfile(file_list: list) -> None:
     :raises: IIBError when the Dockerfile edit is unsuccessful.
     """
     copied_cache_found = False
-    match_str = f'COPY cache /tmp/cache'
+    match_str = f'COPY --chown=1001 cache /tmp/cache'
```
Comment (pbortlov) [inline iib/workers/tasks/opm_operations.py:430]: do you really need it for matching the string? would i match anything?

Comment (yashvardhannanavati) [reply iib/workers/tasks/opm_operations.py:430]: Sorry I didn't get it. The match_str is to make sure the replacement was successful. It's just verifying the correct line was added

Comment (pbortlov) [reply iib/workers/tasks/opm_operations.py:430]: this line is adding "--chown=1001" to be matched in the line below "if match_str in line". Is it a specific case to look for "chown" in dockerfile? my concern is that it won't be matched/find at all.

Comment (yashvardhannanavati) [reply iib/workers/tasks/opm_operations.py:430]: yes, it is a specific case to look for the entire line. If it is not found, an error is raised

Comment (pbortlov) [reply iib/workers/tasks/opm_operations.py:430]: thanks for the response

**Review by pbortlov — APPROVED** [review]

**Pull request merged**